test/testdrive/mzcompose.yml

```diff
@@ -74,34 +74,6 @@ mzworkflows:
         service: testdrive-svc
         command: --ci-output esoteric/pubnub/pubnub.td
 
-  # Run a couple of tests that verify that our http* proxy configuration works
-  ci-proxy:
-    env:
-      ALL_PROXY: http://squid:3128
```

Comment:
It was a bit incorrect to set variables this way, as if a Mz service had been started at that point, any variables set here would not influence that service. So in the python-based test we define dedicated services for each set of env variables under test.